lambdas/src/apis/collections/controllers/collections.ts

```diff
@@ -94,16 +94,23 @@ export async function contentsThumbnail(client: SmartContentClient, req: Request
   // Path: /contents/:urn/thumbnail
   const { urn } = req.params
 
-  // TODO resize
-
   try {
     const entity = await fetchEntity(client, urn)
 
     const wearableMetadata: WearableMetadata = entity.metadata
     const hash = getFileHash(entity, wearableMetadata.thumbnail)
 
-    const headers: Map<string, string> = await client.pipeContent(hash, res as any as ReadableStream<Uint8Array>)
-    headers.forEach((value, key) => res.setHeader(key, value))
+    if (req.query.size) {
```

Comment:
I suggest to use a `/:size/` url param instead of query param to enable edge caching, QueryParameters are hardly cacheable and preferably avoided